### ISSUE TITLE
Instructions for Displaying GUI on VNC Client

### ISSUE THREAD

**Issue by ekaplan7** [jonescompneurolab/hnn#328]
The instructions for running HNN on Oscar seem to be outdated. Instructions read to launch the GUI using the 'singularity run hnn.simg' command however this file is not present after the 'singularity pull' command. I was able to follow the instructions on a previously closed issue: https://github.com/jonescompneurolab/hnn/issues/258.

**Comment by rythorpe** [comment]
This is true, we are not currently maintaining the HNN installation instructions for Oscar.

**Comment by ntolley** [comment]
Should we just update the website to make that clear? I'll look into it if you agree

**Comment by rythorpe** [comment]
Yeah, at least until we get the new GUI up and running.
